--- libmusicbrainz-3.0.2/test/main.cpp.orig	2007-07-10 20:15:23.000000000 +0200
+++ libmusicbrainz-3.0.2/test/main.cpp	2008-12-27 11:02:10.994512992 +0100
@@ -1,10 +1,10 @@
-#include <string>
+//#include <string>
+#include <cppunit/CompilerOutputter.h> 
 #include <cppunit/TestResult.h>
 #include <cppunit/TestResultCollector.h> 
 #include <cppunit/TestRunner.h>
 #include <cppunit/extensions/TestFactoryRegistry.h>
 #include <cppunit/BriefTestProgressListener.h> 
-#include <cppunit/CompilerOutputter.h> 
 
 int 
 main(int argc, char **argv)
